src/modules/appUI.js

```diff
@@ -12,16 +36,31 @@ const filmTemplate = (info) => `
           <i class="far fa-heart"></i>
         </div>
       </div>
-      <button type="button" class="comments">Comments</button>
+      <button type="button" class="comments" id="${info.id}">Comments</button>
     </li>
   `;
-
-const displayMovies = () => {
+const displayMovies = async () => {
   const filmsContainer = document.querySelector('.films');
+
   filmsContainer.innerHTML = '';
-  getMovies().forEach(async (movie) => {
+  const movieList = getMovies().map(async (movie, index) => {
     const movieData = await movie;
     filmsContainer.innerHTML += filmTemplate(movieData);
+    const btnComment = document.querySelectorAll('.comments');
+    btnComment.forEach((x) => {
+      x.addEventListener('click', async () => {
+        const resp = await fetch(
+          `https://api.tvmaze.com/lookup/shows?thetvdb=${x.id}`
+        );
+        const data = await resp.json();
+        console.log(data);
```

Comment:
I also suggest that we remove console logs from our committed code. According to [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), it’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.